src/main/java/redis/clients/jedis/BinaryJedis.java

```diff
@@ -95,6 +95,14 @@ public BinaryJedis(final String host, final int port, final int connectionTimeou
     client.setSoTimeout(soTimeout);
   }
 
+  public BinaryJedis(final String host, final int port, final int connectionTimeout,
+      final int soTimeout, final String password) {
```

Comment:
There isn't any other constructor with direct password param. The constructor with URI accepts password and does auth. I'm not so sure what would be better here also considering inclusion of client name. IMHO, remove this constructor for the time being.